src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,22 @@
 import React from 'react';
+import { Comment } from '../../type/types';
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+interface Props {
+  comments: Comment[];
+}
+
+export const CommentList: React.FC<Props> = ({ comments }) => (
+  <ul className="CommentList">
+    {comments.map((text) => {
```

Comment:
I think it would be more logical to chose some other name for this "text". cause actually it's not text but single comment

Reply:
agree, let's fix it